### PR TITLE
Copy job deadline exceeded

### PR DESCRIPTION
**Pull request by przemyslaw-jasinski**

### CODE REVIEW

File: src/commons/big_query/big_query_job_error.py [hunk 1]
```diff
@@ -32,10 +34,13 @@ def should_be_retried(self):
         else:
             raise self.bq_error
 
+    def is_deadline_exceeded(self):
+        return self.bq_error.resp.status == 500 and \
+               self.full_reason.find('Deadline exceeded') != -1
```
Comment (marcin-kolda) [inline src/commons/big_query/big_query_job_error.py]: `and 'Deadline exceeded' in self.full_reason`

File: src/commons/big_query/copy_job_async/copy_job/copy_job_service.py [hunk 2]
```diff
@@ -54,18 +55,21 @@ def __schedule(source_big_query_table, target_big_query_table, job_id,
...
+            job_reference = BigQuery().insert_job(target_project_id, job_data)
             logging.info("Successfully insert: %s", job_reference)
             return job_reference
         except HttpError as bq_error:
             copy_job_error = BigQueryJobError(bq_error,
                                               source_big_query_table,
                                               target_big_query_table)
-            if copy_job_error.should_be_retried():
+            if copy_job_error.is_deadline_exceeded():
+                job_json = CopyJobService.__get_job(job_id, target_project_id,
```
Comment (marcin-kolda) [inline src/commons/big_query/copy_job_async/copy_job/copy_job_service.py]: What will happen when you'll get 404 here?

Comment (przemyslaw-jasinski) [reply src/commons/big_query/copy_job_async/copy_job/copy_job_service.py]: Discussed offline

File: src/commons/big_query/big_query_job_error.py (hunk 1 above)
Comment (przemyslaw-jasinski) [reply src/commons/big_query/big_query_job_error.py]: Done :)

**Pull request merged**